src/components/WidgetHugger.vue

```diff
@@ -272,8 +277,8 @@ const temporaryPosition = computed(() => {
 })
 
 const sizeStyle = computed(() => ({
-  width: `${100 * size.value.width}%`,
-  height: `${100 * size.value.height}%`,
+  width: disableResponsiveness.value ? `${1000 * size.value.width}px` : `${100 * size.value.width}%`,
+  height: disableResponsiveness.value ? `${1000 * size.value.height}px` : `${100 * size.value.height}%`,
```

Comment:
Curiosity: what was the motive for that the custom widgets to be fundamentally different in the resize behavior? I found it to be counterintuitive to the user.